lib/report-portal-client.js

```diff
@@ -53,7 +53,8 @@ class RPClient {
     });
     this.statistics = new Statistics(EVENT_NAME, agentParams);
     this.launchUuid = '';
-    this.nonRetriedItemMap = new Map();
+    this.executableItemMap = new Map();
```

Comment:
I suggest to name it with retry mention, f.e. `itemRetriesChainMap`.
Pls update all subsequent entities accordingly.